server/pages/index.js

```diff
@@ -629,6 +629,10 @@ module.exports = function() {
 		res.redirect( 301, newRoute );
 	} );
 
+	app.get( '/domains', function( req, res ) {
+		res.redirect( '/start/domain-first' );
+	} );
```

Comment:
If we're going to redirect all traffic (both logged in and out), 

1. I'd recommend taking out this line
2. landing this PR (so the flow is available)
3. Add a redirect 301 permanent in nginx see pMz3w-8YM-p2 as an example

Reply:
I'm not a fan of `nginx` redirects. This is something that might change, and I'd rather not bother every time I need to update it. Also makes more sense to me, as the end goal (some nice day), would be to have the landing page in Calypso under `/domains`.

Reply:
Let's at least set a 301 permanent or 302 temporary (if we intend to change this soon) on the response. Note that we shouldn't necessarily consider nginx redirects as set in stone. We just need to communicate changes.

Reply:
```js
app.get( '/domains', function( req, res ) {
	res.redirect( 301, '/start/domain-first' );
} );
```

can be shortened to:
```js
app.get( '/domains', '/start/domain-first' );
```

(it'll default to 301)